syntax = "proto2";

// proto 10002
message change_scene_req
{
    required bytes scene_name = 1;
};

// proto 20002
message change_scene_rsp
{
    required uint32 ret = 1;
    required bytes  scene_name = 2;
};
